tests/conftest.py

```diff
@@ -148,23 +146,51 @@ def __init__(self, *args, **kwargs):
         # Makes it easier to work with the bot in tests
         self._unfreeze()
 
+    # Here we override get_me for caching because we don't want to call the API repeatedly in tests
+    async def get_me(self, *args, **kwargs):
+        if self._bot_user is None:
+            self._bot_user = make_mock_user(self.token)
+        return self._bot_user
+
```

Comment:
can this be moved to a common base class or at least a separate function that both classes call?

Reply:
i'll go with a separate function, the base class may cause some issues in tests